node/tests/lib/deploy-utils.spec.js

```diff
@@ -45,6 +53,19 @@ describe('deploy utils', () => {
     });
   });
 
+  describe('skip automatic state refresh', () => {
+    it.each`
+      params                              | sentParams
+      ${{ [SKIP_STATE_REFRESH]: 'true' }} | ${{ params: { [SKIP_STATE_REFRESH]: 'true' } }}
+      ${{}}                               | ${{ params: {} }}
```

Comment:
could use `{ params }`, no need for another sentParams variable that is used once in that manner

Reply:
Done: 36cdd6b6a2cb7d845c29dd480ff7dfa9516b16fd